lib/components/scanner/scanner_widget.dart

```diff
@@ -1,92 +0,0 @@
-import 'package:flutter/material.dart';
-import 'package:flutter_bloc/flutter_bloc.dart';
-import 'package:qr_code_scanner/qr_code_scanner.dart';
-import 'package:seeds/components/scanner/interactor/scanner_bloc.dart';
-import 'package:seeds/components/scanner/interactor/viewmodels/scanner_events.dart';
-import 'package:seeds/components/scanner/interactor/viewmodels/scanner_state.dart';
-import 'package:seeds/components/scanner/seeds_qr_code_scanner_widget.dart';
-import 'package:seeds/utils/string_extension.dart';
-
-/// Scanner SCREEN
-class ScannerWidget extends StatefulWidget {
-  final ScannerBloc _scannerBloc = ScannerBloc();
-  final ValueSetter<String> resultCallBack;
-
-  ScannerWidget({Key? key, required this.resultCallBack}) : super(key: key);
-
-  void scan() => _scannerBloc.add(Scan());
-
-  void showLoading() => _scannerBloc.add(ShowLoading());
-
-  void stop() => _scannerBloc.add(Stop());
```

Comment:
TBH Neither of these methods are necessary.

Ideally components like this must be **completely standalone.**

Event the **Scan** event to fire the scanning again you only need to rebuild this ScannerScreen (**its initial state is scan**)